test/jasmine/tests/toimage_test.js

```diff
@@ -0,0 +1,112 @@
+// move toimage to plot_api_test.js
+//  once established and confirmed?
+
+var Plotly = require('@lib/index');
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+var subplotMock = require('@mocks/multiple_subplots.json');
+
+
+describe('Plotly.toImage', function() {
+    'use strict';
+    var gd;
+
+    beforeEach(function() {
+        gd = createGraphDiv();
+    });
+
+    afterEach(destroyGraphDiv);
+
+    it('should be attached to Plotly', function() {
+        expect(Plotly.toImage).toBeDefined();
+    });
+
+    it('should return a promise', function(done) {
+        function isPromise(x) {
+            return !!x.then || typeof x.then === 'function';
```

Comment:
I think you may have meant to use `&&` in which case, only the second clause is required.